packages/core/src/useFormField/useFormField.ts

```diff
@@ -477,13 +115,38 @@ export type ExposedField<TValue> = {
    * @param mutate - Whether to set errors on the field as a result of the validation call, defaults to `true`.
    */
   validate: (mutate?: boolean) => Promise<ValidationResult>;
+
+  /**
+   * Props for the label element.
+   */
+  labelProps: Ref<AriaLabelProps>;
+
+  /**
+   * Props for the description element.
+   */
+  descriptionProps: Ref<AriaDescriptionProps>;
+
+  /**
+   * Props for the error message element.
+   */
+  errorMessageProps: Ref<AriaErrorMessageProps>;
+
+  /**
+   * The field instance.
+   */
+  _field: FormField<TValue>;
 };
 
+export function useFormFieldContext<TValue = unknown>() {
+  return inject(FormFieldKey, null) as FormField<TValue> | null;
+}
+
 export function exposeField<TReturns extends object, TValue>(
   obj: TReturns,
   field: FormField<TValue>,
 ): ExposedField<TValue> & TReturns {
-  const exposedField = {
+  return {
+    _field: markRaw(field),
```

Comment:
@logaretm Do we need to expose `_field` to the user?

I'm thinking that it can be confusing as most of the `_field` properties are already exposed.

Would there be any benefit to be able to extract a property from both?

```ts
const { isTouched, _field } = useTextField({ label: 'label' })

const v1 = isTouched.value
const v2 = _field.isTouched.value

```